felles/src/main/kotlin/no/nav/familie/kontrakter/felles/ef/PeriodeOvergangsstønad.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.familie.kontrakter.felles.ef
+
+import java.time.LocalDate
+
+/**
+ * Klasser som brukes for å hente perioder for overgangsstønad fra enslig forsørger (ef-sak)
+ */
+
+data class PerioderOvergangsstønadRequest(val ident: String,
+                                          val fomDato: LocalDate? = null,
+                                          val tomDato: LocalDate? = null)
+
+data class PerioderOvergangsstønadResponse(val perioder: List<PeriodeOvergangsstønad>)
+
+data class PeriodeOvergangsstønad(val ident: String,
+                                  val fomDato: LocalDate,
+                                  val tomDato: LocalDate,
+                                  val fullOvergangsstønad: Boolean,
```

Comment:
fullOvergangsstønad, denne skal ikke brukes av Arena, men av Barnetrygd.
Men slik det er akkurat nå så får jeg ikke satt denne til riktig verdi. Så lurer på om jeg skal sette denne til nullable. Eller slette den, å legge til den senere når barnetrygd trenger den (har møte med de i neste uke men ønsker att få merget denne slik att jeg kan få laget pr for de andre applikasjonene)